README.md

```diff
@@ -115,6 +115,51 @@ If all your [browser targets](https://guides.emberjs.com/release/configuring-emb
 
 The way you do import remains same.
 
+### Error Handling
+
+`ember-fetch` comes with utility functions for matching responses and their response status codes.
+These utilities are to assist when transitioning from [`ember-ajax`](https://github.com/ember-cli/ember-ajax#error-handling)
+These include
```

Comment:
Can you explain more about fetch responses and errors? Maybe something like

---

A `fetch` response is successful if `response.ok` is true, otherwise you can read the status code to determine the bad response type. `fetch` will only reject with [network errors](https://fetch.spec.whatwg.org/#concept-network-error). 

`ember-fetch` provides some utils:

- `isBadRequestResponse` (400)
- `isUnauthorizedResponse` (401)
- `isForbiddenResponse` (403)
- `isNotFoundResponse` (404)
- `isConflictResponse` (409)
- `isGoneResponse` (410)
- `isInvalidResponse` (422)
- `isServerErrorResponse` (5XX)
- `isAbortError` [aborted network error](https://developers.google.com/web/updates/2017/09/abortable-fetch)

An example:

```js
import Route from '@ember/routing/route';
import fetch from 'fetch';
import {
  isAbortError,
  isUnauthorizedResponse,
  isServerErrorResponse
} from 'ember-fetch/errors';
export default Route.extend({
  async model() {
    try {
      let response = await fetch('/omg.json');
      if (response.ok) {
        return response.json();
      } else if (isUnauthorizedResponse(response)) {
        // handle 401 response
      } else if (isServerErrorResponse(response)) {
        // handle 5xx respones
      }
    } catch (error) {
      if (isAbortError(error)) {
        // handle aborted network error
      }
      // handle network error
    }
  }
});
```

Reply:
I just checked the guide https://guides.emberjs.com/v3.10.0/routing/specifying-a-routes-model/#toc_fetch-example is using promise style, so I added 1 more commit to align with that.

Reply:
Ah great! Thanks for that. I think that's a good idea